src/ui/component/stepper/stepperProgress/progressBar/progressBar.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames'
+import './progressBar.scss'
+
+export type ProgressBarColor = 'grey' | 'blue' | 'green'
```

Comment:
Instead of coupling the `ProgressBar` behavior with style considerations (`color`), I would have handled the `status` as a prop that is then taken into account in the component's stylesheet.